### PR TITLE
Allow for provisioner-cli to be run from anywhere

### PR DESCRIPTION
**Pull request by jtumber-ibm**
Fixes: #950

### CODE REVIEW

**Review by huoqifeng — COMMENTED** [review]
Thank you @jtumber-ibm to pick it up, basically, it looks good to me, but I'm not aware is there a better solution, I'll add more reviewer for more comments here.

**Review by yoheiueda — COMMENTED** [review]
I think it is more flexible if we can make it an optional argument, and also configurable with an environment variable and an entry in a provision properties file.

File: test/provisioner/provision_azure.go [hunk 1]
```diff
@@ -375,8 +376,8 @@ func isAzureKustomizeSecretKey(key string) bool {
 	}
 }
 
-func NewAzureInstallOverlay() (InstallOverlay, error) {
-	overlay, err := NewKustomizeOverlay("../../install/overlays/azure")
+func NewAzureInstallOverlay(installDir string) (InstallOverlay, error) {
+	overlay, err := NewKustomizeOverlay(path.Clean(fmt.Sprintf("%s/overlays/azure", installDir)))
```
Comment (yoheiueda) [inline test/provisioner/provision_azure.go:380]: I think we can simply write this line like this.
```
NewKustomizeOverlay(filepath.Join(installDir, "overlays/azure"))
```

File: test/tools/README.md [hunk 2]
```diff
@@ -26,10 +26,12 @@ export LOG_LEVEL=${LOG_LEVEL}
 export CLOUD_PROVIDER=${CLOUD_PROVIDER}
 export TEST_PROVISION_FILE=${PROPERTIES_FILE_PATH}
 export TEST_PROVISION="yes"
-./caa-provisioner-cli -actions=${ACTION}
+./caa-provisioner-cli -actions=${ACTION} ${INSTALL_DIR}
```
Comment (liudalibj) [inline test/tools/README.md:29]: Can we follow the same style as others, use an ENV var at here?
```
export INSTALL_DIR=${INSTALL_DIR}
```

**Review by liudalibj — APPROVED** [review]
LGTM thanks @jtumber-ibm

**Review by huoqifeng — APPROVED** [review]
/lgtm

**Pull request merged**